css-three-column-layout/styles.css

```diff
@@ -0,0 +1,57 @@
+body {
+  background: #1896d9;
+  font-family: Lato, sans-serif;
+  color: white;
+}
+
+.container {
+  max-width: 1100px;
+  margin: 30px auto 0;
+}
+
+.row {
+  display: flex;
+  justify-content: space-around;
+}
```

Comment:
You would want to move this `justify-content: space-around` into it's own class. Your `row` should only do one job which sets the `display: flex` and by default, makes the `flex-direction: row`